### PR TITLE
Unit test contact provider registration

### PR DESCRIPTION
**Pull request by DeepDiver1975**
@rullzer @PVince81 @icewind1991 @schiesbn @blizzz please review and test - THX

### CODE REVIEW

**Comment by icewind1991** [comment]
:+1: looks good

**Comment by DeepDiver1975** [comment]
@LukasReschke @MorrisJobke please have a look - THX

File: apps/dav/appinfo/application.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<?php
```
Comment (schiesbn) [inline apps/dav/appinfo/application.php]: missing license header

**Comment by schiesbn** [comment]
beside the missing license header it looks good :+1:

File: apps/dav/appinfo/application.php (hunk 1 above)
Comment (DeepDiver1975) [inline apps/dav/appinfo/application.php]: I'll rerun our license header tool in a pr of it's own

**Pull request merged**